docs/features/software-catalog/well-known-annotations.md

```diff
@@ -70,6 +70,25 @@ The value of this annotation is a location reference string (see above). If this
 annotation is specified, it is expected to point to a repository that the
 TechDocs system can read and generate docs from.
 
+### backstage.io/browser-view-url, backstage.io/browser-edit-url, backstage.io/browser-source-url
+
+```yaml
+# Example:
+metadata:
+  annotations:
+    backstage.io/browser-view-url: https://some.website/catalog-info.yaml
+    backstage.io/browser-edit-url: https://github.com/my-org/catalog/edit/master/my-service.jsonnet
+    backstage.io/browser-source-url: https://github.com/my-org/my-service
```

Comment:
@backstage/maintainers, thoughts on there? I'm thinking we drop `browser-`, e.g. `backstage.io/view-url`?

Full example:

```yaml
backstage.io/view-url: https://some.website/catalog-info.yaml
backstage.io/edit-url: https://github.com/my-org/catalog/edit/master/my-service.jsonnet
backstage.io/source-url: https://github.com/my-org/my-service
```

Reply:
I'd prefer the shorter version FWIW. I blindly copied the browser prefix from the issue but on reflection it only really makes sense for `edit` since that link literally edits a file in your browser.

Reply:
Yeah I think you are right. In particular the source one should be usable by other plugins as a proper pointer to the location of the source that the entity describes. I think it should even be an actual location, and named `source-location` symmetrically to the `managed-by-location`.

```yaml
backstage.io/source-location: url:https://github.com/my-org/my-service
```

As a side note, I think it's time we standardized on a location string parser that's smart enough to interpret `http(s):` as `url:http(s):`... :)

Reply:
If we go with my suggestion on the latter (@Rugvip?) then maybe this section of the docs could be split apart into the view-url and edit-url in one heading, and the source-location in another.

Reply:
@freben @Rugvip I made that change to a proper location for source files in https://github.com/backstage/backstage/pull/4575/commits/e0fc8db66a79d5f236ef251b2effcc2b690b5f8c, can be reverted if we change our minds.